### PR TITLE
chore(docs): Add missing client metrics docs

### LINKED ISSUE #4057: Add missing data points to client side metrics documentation
We are missing a lot of data points here in our definition document which makes navigating data dog difficult. We need to refresh it an define new metrics.

https://github.com/mozilla/fxa-content-server/blob/master/docs/client-metrics.md

### PR DESCRIPTION
**Pull request by vbudhram**
Bringing our client metrics doc up to speed with new items. Added confirm_signin and complete_signin and some others.

Fixes #4057

### CODE REVIEW

File: docs/client-metrics.md [hunk 1]
```diff
@@ -86,62 +86,73 @@ The event stream is a log of events and the time they occurred while the user is
...
-* error.complete_reset_password.auth.1026 - User clicked on a damaged verification link.
 
-#### complete_sign_up
-* complete_sign_up.verification.success - email successfully verified.
-* complete_sign_up.resend - A verification email was resent after an expired link was opened.
-* error.complete_sign_up.auth.1025 - User clicked on an expired verification link.
-* error.complete_sign_up.auth.1026 - User clicked on a damaged verification link.
-* error.complete_sign_up.auth.1040 - User tried to verify an account that does not exist.
-* error.complete_sign_up.auth.1041 - User tried to verify a sign-in that has already been verified.
+#### verify_email
```
Comment (vbudhram) [inline docs/client-metrics.md]: @shane-tomlinson in datadog `complete_sign_up` does not appear, seems like it has been replaced with `verify_email`?

**Comment by vbudhram** [comment]
@shane-tomlinson @vladikoff Mind an r?

File: docs/client-metrics.md (hunk 1 above)
Comment (shane-tomlinson) [inline docs/client-metrics.md]: Oh, yeah, my mistake probably. Thanks for fixing the name.

File: docs/client-metrics.md [hunk 2]
```diff
@@ -86,62 +86,73 @@ The event stream is a log of events and the time they occurred while the user is
...
+* error.cookies_disabled.auth.1003 - Cookies are still disabled.
+
 #### delete_account
-* error.delete-account.auth.121 - account locked
-* delete-account.unlock-email.send - user attempted to send unlock email
-* delete-account.unlock-email.send.success - unlock email successfully sent
+* error.delete_account.auth.1009 - password must be at least 8 characters
 * delete-account.deleted - user successfully deleted an account
 
 #### force_auth
```
Comment (shane-tomlinson) [inline docs/client-metrics.md]: Can you remove `illegal_iframe` below?

**Pull request merged**